### PR TITLE
fix(RELEASE-1082): linting issues in push-rpm-manifests-to-pyxis

### PR DESCRIPTION
**Pull request by johnbieren**
## Checklist before requesting a review
- [ ] I have marked as draft or added `do not merge` label if there's a dependency PR
  - If you want reviews on your draft PR, you can add reviewers or add the `release-service-maintainers` handle if you are unsure who to tag
- [ ] My commit message includes `Signed-off-by: My name <email>`
- [ ] I have bumped the task/pipeline version string and updated changelog in the relevant README
- [ ] I read CONTRIBUTING.MD and [commit formatting](CONTRIBUTING.md#commit-message-formatting-and-standards)

### CODE REVIEW

File: tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml [hunk 1]
```diff
@@ -176,21 +175,21 @@ spec:
           $UPLOAD_SCRIPT --retry --image-id "$IMAGEID" --sbom-path "$FILE" --verbose > "${IMAGEID}.out" 2>&1 &
 
           jobs+=($!)  # Save the background process ID
-          images+=($IMAGEID)
+          images+=("$IMAGEID")
           ((++count))
 
-          if [ $((count%N)) -eq 0 -o $((count)) -eq $total ]; then
```
Comment (mmalina) [inline tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml:182]: Why didn't it like this? I'm curious. If you remember. I thought this was perfectly ok.

**Review by mmalina — DISMISSED** [review]

File: tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml (hunk 1 above)
Comment (johnbieren) [reply tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml:182]: https://github.com/koalaman/shellcheck/wiki/SC2166

Comment (mmalina) [reply tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml:182]: Interesting, thanks.

**Review by mmalina — APPROVED** [review]

**Pull request merged**